sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -11,6 +13,47 @@
 LCClient = noaa.NOAAIndicesClient()
```

Comment:
Can you add a new line here to break this from the import statements.

Reply:
Sure